Scripts/dea_classificationtools.py

```diff
@@ -1214,40 +1234,39 @@ def spatial_train_test_split(X, y, coordinates, cluster_method, kfold_method,
     **kwargs : optional,
         Additional keyword arguments to pass to sklearn.cluster.Kmeans or
         sklearn.mixture.GuassianMixture depending on the cluster_method argument.
-
     Returns
     -------
     Tuple :
         Contains four arrays in the following order:
             X_train, X_test, y_train, y_test
-
     """
     if kfold_method == 'SpatialShuffleSplit':
```

Comment:
Consider throwing a warning if you use train_size/test_size rather than n_splits, especially if you're considering fixing it in future